src/erlcloud_ddb2.erl

```diff
@@ -1369,7 +1369,7 @@ batch_write_item(RequestItems, Opts, Config) ->
 %%% CreateTable
 %%%------------------------------------------------------------------------------
```

Comment:
Line 1370 is too long: %%%------------------------------------------------------------------------------.